addons/mail/static/src/components/composer_text_input/composer_text_input.js

```diff
@@ -153,6 +153,12 @@ class ComposerTextInput extends Component {
      * @private
      */
     _update() {
+        // Needed to ensure the right composer is focused when the composer changes.
+        // For example when changing channel in discuss component is reused but composer model
+        // changes so the textarea is still focused but associated composer model is not
```

Comment:
Would put comment inside the `if`. More natural, and it makes it less important to understand the rest of `_update()`.